tests/foreman/cli/test_remoteexecution.py

```diff
@@ -551,7 +551,6 @@ def test_positive_run_reccuring_job(self, rex_contenthost):
             }
         )
         result = JobInvocation.info({'id': invocation_command['id']})
-        assert_job_invocation_status(invocation_command['id'], client.hostname, 'queued')
```

Comment:
Why do we no longer want to assert `queued` here? I see another test in this file that still does this.